### DISCUSSION TITLE
WITSML to LAS

### DISCUSSION THREAD

**Discussion by issimo007** [kinverarity1/lasio#533] [Ideas]
Hello has anyone every tried to read witsml data (xml formal) and convert it to an LAS file?

**Comment by kinverarity1** [comment]
I haven't ever tried but these links might help you:

Convert WITSML file to DataFrame:
- [WITSML import via Pandas and Python (blog post)](https://frombitumentobinary.com/witsml-via-pandas-python.html)

Convert DataFrame to LAS file:
- [Building a LAS file from scratch (lasio documentation)](https://lasio.readthedocs.io/en/stable/building.html)
- [``lasio.LASFile.set_data_from_df`` (lasio documentation)](https://lasio.readthedocs.io/en/stable/lasio.html#lasio.LASFile.set_data_from_df)
